README.md

```diff
@@ -63,6 +63,9 @@
   <a href="http://taskforce.sh"  style="margin-left: 50px;">
     <img src="http://taskforce.sh/assets/logo_square.png" width="160" alt="Taskforce.sh, Inc" />
   </a>
+  <a href="https://getstream.io">
+    <img src="https://i.imgur.com/asDSMwp.png" alt="GetStream.io" />
+  </a>
```

Comment:
does not seem like the url is the correct one.
![image](https://user-images.githubusercontent.com/95200/40085349-6b975674-589a-11e8-82b7-0cf04482d725.png)